sdk/cognitiveservices/Azure.OpenAI.Inference/tests/Azure.OpenAI.Inference.Tests.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>$(RequiredTargetFrameworks);net47</TargetFrameworks>
+  </PropertyGroup>
+  
+    <ItemGroup>
+    <PackageReference Include="Microsoft.CSharp" VersionOverride="4.7.0" />
```

Comment:
We should not be overriding versions.   Can you help me understand the need to deviate from the central versions?